sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -123,7 +123,6 @@ See the full details regarding [authentication][cognitive_authentication] of cog
 
  - Recognizing form fields and content using custom models trained to recognize your custom forms. These values are returned in a collection of `RecognizedForm` objects.
  - Recognizing form content, including tables, lines and words, without the need to train a model. Form content is returned in a collection of `FormPage` objects.
- - Recognizing common fields from US receipts, using a pre-trained receipt model on the Form Recognizer service. These fields and meta-data are returned in a collection of `RecognizedReceipt` objects.
```

Comment:
I'd still include this sentence, just say that it returns a collection of `RecognizedForm`